src/modules/mc_hover_thrust_estimator/MulticopterHoverThrustEstimator.cpp

```diff
@@ -166,10 +161,24 @@ void MulticopterHoverThrustEstimator::Run()
 
 		_hover_thrust_ekf.predict(dt);
 
-		vehicle_local_position_setpoint_s local_pos_sp;
+		vehicle_attitude_s vehicle_attitude{};
+		_vehicle_attitude_sub.copy(&vehicle_attitude);
```

Comment:
TODO: use all of these messages (vehicle_local_position, vehicle_attitude, vehicle_thrust_setpoint)  on the same timestamp_sample?

Reply:
Given that we don't take the motor lag into account (the model assumes instant acceleration), I don't think that aligning the data perfectly is important.

Reply:
Given that we don't take the motor lag into account (the model assumes instant acceleration), I don't think that aligning the data perfectly is important.